backend/app/api/routes/soundfonts.py

```
@@ -0,0 +1,131 @@
+    display_name: str
+    category: str
+    description: Optional[str]
+    storage_key: str
+    created_at: datetime
+    updated_at: datetime
+
+@router.get("/public", response_model=List[dict])
+async def get_public_soundfonts(
+    request: Request,
+    category: Optional[str] = None,
+    current_user: dict = Depends(get_current_user)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)